.github/workflows/centos-stream-9.yml

```diff
@@ -690,3 +690,88 @@ jobs:
           path: |
             *.json
             *.log
+
+  pre-edge-vsphere-cs9:
+    needs: pr-info
+    if: ${{ needs.pr-info.outputs.allowed_user == 'true' && github.event.issue.pull_request &&
+            (endsWith(github.event.comment.body, '/test-cs9') ||
+            endsWith(github.event.comment.body, '/test-cs9-vsphere')) }}
+    runs-on: ubuntu-latest
+    env:
+      STATUS_NAME: edge-vsphere-installer-cs9
+
+    steps:
+      - name: Create in-progress status
+        uses: octokit/request-action@v2.x
+        with:
+          route: 'POST /repos/${{ github.repository }}/statuses/${{ needs.pr-info.outputs.sha }}'
+          context: ${{ env.STATUS_NAME }}
+          state: pending
+          description: 'Runner has been deploying...'
+          target_url: 'https://github.com/${{ github.repository }}/actions/runs/${{ github.run_id }}'
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  edge-vsphere-cs9:
+    needs: [pr-info, pre-edge-vsphere-cs9]
+    runs-on: [kite, x86_64, rhos-01, centos-stream-9, medium]
+    env:
+      STATUS_NAME: edge-vsphere-installer-cs9
```

Comment:
What does the `installer` mean in STATUS_NAME. I don't think it's reasonable.

Reply:
removed installer